### ISSUE TITLE
Loop & Custom Queries not working in front-page.php

### ISSUE THREAD

**Issue by venuscrush** [wprig/wprig#600]
## Issue Overview
Trying to customize the Wordpress loop in my front-page.php file. It seems as though this is not possible as it has worked in the previous wprig as well as underscores starter theme.

## Describe your environment
I'm using the current wprig setup + VSC, as instructed by Morten (remarkable instructor).

## Steps to Reproduce

		<?php
		$args     = array(
			'post_type'      => 'post',
			'posts_per_page' => '6',
			 'orderby'        => 'rand',
		);
		$my_query = new WP_Query( $args );
		if ( $my_query->have_posts() ) {
			while ( $my_query->have_posts() ) {
				$my_query->the_post();

				get_template_part( 'template-parts/content/entry_cards' );

			} // end while
		} // end if
		?>

## Expected Behavior
I would like to randomly display 6 posts that are styled according to my custom template part (entry_cards).

## Current Behavior
The entire site is not blank, but all I see (using the code above) is a blank area in place of where the posts should be.

## Possible Solution
I'm at a loss as I've been trying to figure this out for several hours to no avail. Any help would be greatly appreciated!

## Screenshots / Video
N/A

## Related Issues and/or PRs
N/A

**Comment by grahammtbr** [comment]
Hey, something I can answer! I just ran into this a few days ago, and figured it out the hard way. Because WP_Query is a class, you need to include "use WP_Query;" in your template. So now your front-page.php should look something like this:
```
<?php
/**
 * Render your site front page, whether the front page displays the blog posts index or a static page.
 *
 * @link https://developer.wordpress.org/themes/basics/template-hierarchy/#front-page-display
 *
 * @package wp_rig
 */

namespace WP_Rig\WP_Rig;
use WP_Query;

get_header();

etc....
```

**Comment by justlevine** [comment]
An alternative to what @grahammtbr suggested:
Instead of (possibly a bunch of) `use` statements, you can throw a `\` before the class you're instantiating. `$var = new \{ClassName( );`.

e.g. `$my_query = new \WP_Query($args);`
or `new \WC_Order($args`)`

**Tl;Dr** anytime youre in a namespaced file and need to use `new {ClassName()}` of something that isnt part of your theme, you either should add a `use {ClassName};` to the top of your file, or just call it with '\'.

**Comment by grahammtbr** [comment]
> Instead of (possibly a bunch of) `use` statements, you can throw a `\` before the class you're instantiating. `$var = new \{ClassName( );`.

And I learnt something new!

**Comment by venuscrush** [comment]
Excellent—works like a charm! Thank you both for all your support. Much appreciated.

**Comment by justlevine** [comment]
If its solved, remember to close the issue :-)
